packages/tbds-support/lib/breakpoints.scss

```diff
@@ -0,0 +1,21 @@
+$tbds-breakpoint-small: 32em !default;
+$tbds-breakpoint-medium: 50em !default;
+$tbds-breakpoint-large: 72em !default;
+
+@mixin small {
```

Comment:
We should namespace each mixin name with `tbds-`, to denote that it’s coming from the design system and to prevent conflicts with consumer apps which may have a mixin of the same name.

I believe if you rebase this branch, Hound/stylelint should warn you about this, as I added a lint for mixin names in https://github.com/thoughtbot/design-system/commit/0af82493cb364b49c08bedf7918f4b85783a2037